### PR TITLE
(test): Add tests for `overflow-menu.component.tsx`

### PR DESCRIPTION
**Pull request by ayush-AI**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

In this PR I have added tests for `overflow-menu.component.tsx`

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by ayush-AI** [comment]
cc @Piumal1999 @anjula-sack @jayasanka-sack

File: packages/esm-patient-search-app/src/patient-search-page/patient-banner/ui-components/overflow-menu.test.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    // expect(menuOptions).toBeNull();
+
+    fireEvent.click(menuButton);
+
+    const updatedMenuOptions = screen.getByText('Option 1');
+    expect(updatedMenuOptions).toBeInTheDocument(); // Menu should be open after click
+
+    fireEvent.click(menuButton);
+
+    // expect(screen.queryByText('Option 1')).toBeNull();
```
Comment (anjula-sack) [inline packages/esm-patient-search-app/src/patient-search-page/patient-banner/ui-components/overflow-menu.test.tsx:26]:
```suggestion
```

File: packages/esm-patient-search-app/src/patient-search-page/patient-banner/ui-components/overflow-menu.test.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    expect(screen.getByRole('button', { name: 'Test Menu' })).toBeInTheDocument();
+  });
+
+  it('should toggle the menu when the button is clicked', () => {
+    render(<CustomOverflowMenuComponent menuTitle="Test Menu" dropDownMenu={true} children={<li>Option 1</li>} />);
+
+    const menuButton = screen.getByRole('button', { name: 'Test Menu' });
+    const menuOptions = screen.queryByText('Option 1');
+
+    // expect(menuOptions).toBeNull();
```
Comment (anjula-sack) [inline packages/esm-patient-search-app/src/patient-search-page/patient-banner/ui-components/overflow-menu.test.tsx:17]:
```suggestion
```

**Review by anjula-sack — DISMISSED** [review]

**Review by denniskigen — APPROVED** [review]
LGTM. Thanks, @ayush-AI

**Review by denniskigen — APPROVED** [review]
LGTM. Thanks, @ayush-AI
